app/assets/javascripts/darkswarm/controllers/authentication/signup_controller.js.coffee

```diff
@@ -9,9 +9,6 @@ Darkswarm.controller "SignupCtrl", ($scope, $http, $window, $location, Redirecti
 
   $scope.submit = ->
     $http.post("/user/spree_user", {spree_user: $scope.spree_user}).success (data)->
-       if Redirections.after_login
-        $window.location.href = $window.location.origin + Redirections.after_login
-       else
-        $window.location.href = $window.location.origin + $window.location.pathname  # Strips out hash fragments
```

Comment:
Are we losing this redirect information for users who are signing up? @myriamboure can you make sure you test what happens to users who sign up on a page other than the homepage. eg. checkout. Specifically, where are they redirected after they have confirmed their email address. I suspect homepage? If so, is this behaviour ok? I would prefer them to be redirected to where they originally came from. Especially if they are part way through an order.

Reply:
We are not supposed to loos the redirect information, users are supposed to be sent back to the thing they were originately doing. We will be careful with that on testing.

Reply:
I think the point of removing this was precisely to keep users on the same page, but also to not trigger a page reload. The user isn't logged in automatically on signup as was the case prior to email confirmations, which means this `Redirect.after_login` is not useful. Also, the feedback/response for registration is displayed in the signup form after an asynchronous request, and the view is updated in place.

Reply:
Ok, cool. Thanks for the explanation. I'm happy if testers are.